pkg/synthetictests/historicaldata/disruption_types.go

```diff
@@ -71,48 +65,46 @@ func NewMatcher(historicalJSON []byte, defaultReturn float64) (BestMatcher, erro
 		if err != nil {
 			return nil, err
 		}
-		curr := StatisticalData{
+		curr := DisruptionStatisticalData{
 			DataKey: currDecoded.DataKey,
 			P95:     p95,
 			P99:     p99,
 		}
 		historicalData[curr.DataKey] = curr
 	}
 
-	return &bestMatcher{
+	return &DisruptionBestMatcher{
 		historicalData: historicalData,
-		defaultReturn:  defaultReturn,
 	}, nil
 }
 
-func NewMatcherWithHistoricalData(data map[DataKey]StatisticalData, defaultReturn float64) BestMatcher {
-	return &bestMatcher{
+func NewDisruptionMatcherWithHistoricalData(data map[DataKey]DisruptionStatisticalData) *DisruptionBestMatcher {
+	return &DisruptionBestMatcher{
 		historicalData: data,
-		defaultReturn:  defaultReturn,
 	}
 }
 
-func (b *bestMatcher) BestMatch(name string, jobType platformidentification.JobType) (StatisticalData, string, error) {
+func (b *DisruptionBestMatcher) bestMatch(name string, jobType platformidentification.JobType) (DisruptionStatisticalData, string, error) {
 	exactMatchKey := DataKey{
-		Name:    name,
-		JobType: jobType,
+		BackendName: name,
+		JobType:     jobType,
 	}
 
-	if percentiles, ok := b.historicalData[exactMatchKey]; ok {
+	if percentiles, ok := b.historicalData[exactMatchKey]; ok && percentiles.JobRuns > minJobRuns {
```

Comment:
Will start sending all data over for disruption in https://github.com/openshift/ci-tools/pull/3284, but first origin needs to know to ignore it if under 100 runs.